selenium/Tests/GraphTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using OpenQA.Selenium;
+using System;
+
+namespace SeleniumTests.Tests
+{
+    class GraphTests
+    {
+
+        public static readonly TestCase DeleteSHACLProperty =
+           new TestCase(
+           "SHACL properties can be deleted",
+       (driver, log) =>
+       {
+           driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            //Log in
+            driver.Login();
+           String currentPath = System.IO.Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.FullName;
+           String path = currentPath + @"\testfiles\demo_shacl.ttl";
+            //Open a SHACL file
+            driver.OpenFile(path);
```

Comment:
I see the following getting duplicated over and over again.

```csharp
String currentPath = System.IO.Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.FullName;
String path = currentPath + @"\testfiles\demo_shacl.ttl";
// Open a SHACL file
driver.OpenFile(path);
```

You might want to refactor that pattern into a specialized (extension?) method, so you can just call `driver.OpenTestFile("demo_shacl.ttl");` Also see my comment below w.r.t. the Windows-style paths: you should use `Path.Combine` instead.